### PR TITLE
[FRONT-117] Add tooltips

### PR DESCRIPTION
**Pull request by yinruo**
### **Description of issue**
Add tooltips to some UI components that doesn't have obvious purpose.

### **How I implemented**
npm install react-tooltip
add tooltips to detail page. Tooltips are added to clickable names and icons. Also the icons in the right side bar have tooltips to show what they are.  

### **Other**

### CODE REVIEW

**Review by ArnoClaude — CHANGES_REQUESTED** [review]
Can you put this PR in draft as long as you are still working on it? It just contains a `hello world` tooltip so far.

File: src/components/page/details/RightSidebar.tsx [hunk 1]
```diff
@@ -37,18 +37,36 @@ const RightSidebar = ({ project }: Props) => {
...
-        <Sidebar.Box.GithubStatItem Icon={IssueOpenedIcon} value={project?.issueCount as number} />
-        <Sidebar.Box.GithubStatItem Icon={RepoForkedIcon} value={project?.forkCount as number} />
+        <Sidebar.Box.GithubStatItem
+          Icon={StarIcon}
+          value={project?.starCount as number}
+          id="stars"
+        />
+        <Sidebar.Box.GithubStatItem
+          Icon={IssueOpenedIcon}
+          value={project?.issueCount as number}
```
Comment (DogukanGun) [inline src/components/page/details/RightSidebar.tsx:47]: If project is empty, what happens here ?

**Review by DogukanGun — CHANGES_REQUESTED** [review]

File: src/components/page/details/RightSidebar.tsx
Comment (kigadye) [inline src/components/page/details/RightSidebar.tsx:1]: Capitalize all tooltip values

File: src/components/page/details/RightSidebar.tsx [hunk 2]
```diff
@@ -38,23 +38,46 @@ const RightSidebar = ({ project }: Props) => {
...
+        <Sidebar.Box.GithubStatItem
+          Icon={RepoForkedIcon}
+          value={project?.forkCount as number}
+          id="forks"
+        />
         <Sidebar.Box.GithubStatItem
           IconMetric={<GitHubMetricIcon Icon={RepoForkedIcon} Icon2={PersonIcon} />}
-          value={(project.forkCount || 0) / (project.contributorCount || 1)}
+          value={(project?.forkCount || 0) / (project?.contributorCount || 1)}
+          id="fork/contributor"
```
Comment (kigadye) [inline src/components/page/details/RightSidebar.tsx:59]: Forks/ contributor

File: src/components/page/details/RightSidebar.tsx [hunk 3]
```diff
@@ -38,23 +38,46 @@ const RightSidebar = ({ project }: Props) => {
...
         />
         <Sidebar.Box.GithubStatItem
           IconMetric={<GitHubMetricIcon Icon={IssueOpenedIcon} Icon2={PersonIcon} />}
-          value={(project.issueCount || 0) / (project.contributorCount || 1)}
+          value={(project?.issueCount || 0) / (project?.contributorCount || 1)}
+          id="issue/contributor"
```
Comment (kigadye) [inline src/components/page/details/RightSidebar.tsx:64]: Issues/ contributor

**Review by kigadye — CHANGES_REQUESTED** [review]
Is there a way to style the tooltips to match our design (i.e. gray background color + 5px border radius)?

**Review by kigadye — APPROVED** [review]
Looks great now!

**Review by LordZed400 — APPROVED** [review]

**Pull request merged**